go.mod

```diff
@@ -35,14 +38,12 @@ require (
 	github.com/opencontainers/image-spec v1.0.2-0.20200206005212-79b036d80240 // indirect
 	github.com/openshift/api v3.9.1-0.20190924102528-32369d4db2ad+incompatible
 	github.com/openshift/client-go v0.0.0-20200116152001-92a2713fa240
-	github.com/openshift/custom-resource-status v0.0.0-20200602122900-c002fd1547ca // indirect
 	github.com/openshift/library-go v0.0.0-20200407165825-2e79bd232e72
 	github.com/openshift/oc v0.0.0-alpha.0.0.20200305142246-2576e482bf00
 	github.com/operator-framework/operator-lifecycle-manager v0.0.0-20200422144016-a6acf50218ed
 	github.com/pborman/uuid v1.2.0
 	github.com/pkg/errors v0.9.1
 	github.com/posener/complete v1.1.2
-	github.com/redhat-developer/service-binding-operator v0.3.0
```

Comment:
Instead of removing the dependency, can we try updating this to 0.7.0?

Reply:
Check my description of this PR. We can't update to 0.7.0 because it would require updating OpenShift and Kubernetes dependencies which will require larger code change.

Reply:
My concern is that SBO is in alpha stage right now. Till a GA is out, they can break compatibility. And on the odo side, we're playing catch-up game since a long time now.

My understanding is that they would be doing a GA soon. Correct me if I am wrong. When the GA is out, we can make the changes that makes odo compatible with the _**stable**_ version, no?

If it requires a larger code change then let's do that larger code change. But let's do it by following the SB struct provided by the library. In this case, we're adding a bunch of things to the unstructured struct. The problem with that, IMO, is that it requires understanding of different fields in the SB struct and can't rely on IDE.

Also, this would be setting incorrect precedence, IMO. It'll later pop up as more PRs doing things like this. Today to stay compatible with SBO, tomorrow to stay compatible with `devfile/library`.

If my concern sounds like over-thinking, I'm fine by doing lgtm. Let me know. 😄

Reply:
We need to fix this ASAP. because more and more clusters will update to 0.7.0 and `odo link` is broken there.
If we release odo without this fixed link will be unusable on new clusters. 
I'm working on a larger code change as part of #4553 because that will require this update as well. But this will take a little bit longer and there are a lot of things that can break while doing this. 
This is why I take this approach so we can merge it before 2.1.0 release

Reply:
That makes sense. 👍🏽
LGTM'd.